dependencies.yml

```diff
@@ -131,6 +131,9 @@ dependencies:
   - type: dockerfile
     path: ruby/hanami/Dockerfile
 
+  - type: dockerfile
```

Comment:
good job thinking of that :tada: 

but I think I'll remove this feature (rather having service like that with no configuration required)

Reply:
Should I remove the entry in the dependencies.yml file then?

Reply:
no, I'm just impressed with what you've done :+1: